org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/codemanipulation/AbstractSourceTestCase.java

```diff
@@ -0,0 +1,120 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2008 IBM Corporation and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
```

Comment:
So we're now copying EPL-2 code into a project that hasn't moved to EPL-2 yet (EPL-1 is deprecated, so we should do it eventually). I think this is fine, but would love to get confirmation about it from a professional (@waynebeaton ?)